x/auth/tx/config/depinject.go

```diff
@@ -242,12 +249,17 @@ var (
 )
 
 type AppModule struct {
-	sigVerification ante.SigVerificationDecorator
+	validators []appmodulev2.TxValidator[transaction.Tx]
```

Comment:
I think we shouldn't remove it, but make it extendable as done here: https://github.com/cosmos/cosmos-sdk/pull/21409

Reply:
make sense, let merge it first then I will update this one.

Reply:
Should be merged now 👍🏾